packages/earn-controller/src/EarnController.ts

```diff
@@ -219,18 +281,28 @@ export class EarnController extends BaseController<
   EarnControllerState,
   EarnControllerMessenger
 > {
-  #stakeSDK: StakeSdk | null = null;
+  #earnSDK: EarnSdk | null = null;
 
   #selectedNetworkClientId?: string;
 
-  readonly #stakingApiService: StakingApiService = new StakingApiService();
+  readonly #earnApiService: EarnApiService;
+
+  readonly #addTransactionFn: typeof TransactionController.prototype.addTransaction;
+
+  readonly #supportedPooledStakingChains: number[];
+
+  readonly #env: EarnEnvironments;
 
   constructor({
     messenger,
     state = {},
+    addTransactionFn,
```

Comment:
This new option looks like a breaking change to the constructor, what do you think about documenting it and the new `env` option in the changelog?